### ISSUE TITLE
Multi Encoder

### ISSUE THREAD

**Issue by S-Man42** [S-Man42/GCWizard#65]
(from Jörg via Messenger)

http://fbcs.bplaced.net/multi_encoder_decoder.html

A meta tool which lists the result of many standard encryption in one view.

**Comment by S-Man42** [comment]
This has been requested several times. I am not quite sure about this. There are so many options and tools and whatever. ROT1, ROT2, ROT3, and so on. Hashes, ciphers, binary encodings which lead to ascii and everything backwards... 

But maybe with some common tools it could make sense.

**Comment by t-m-z** [comment]
Yes it is a nice tool, but from a user perspective, up to now I had no need for a meta tool which provides many standard encryption in one view. However, I had the need for a meta tool which many standard decryption in one view. This could ease the solving of riddles in the field. But whats's about the additional data some algortithm needs?
Some encodings like kenny, chicken, duck, pig latin, robber, spoon etc. are self explanatory. So I need no meta tool to identify these.

A solution could be an additional tab "Meta Tool" like favorites. The user can check every function, which should be processed there. That means, the user provides the chiffretext, gcw sets the standard paramter, calculates and provide the out. 
Basically we talk about a large list of "if algorithm is part of the meta tool tab then set the paramters, decrypt, append output".

**Comment by S-Man42** [comment]
The first throw would be to take only ciphers without additional options like those that were mentioned above. If you need any additional options, this could be quite problematic. Maybe the tool should save user inputs which seem to be valid for default inputs. However, that needs a great implementation of #17

**Comment by S-Man42** [comment]
#282 Done.
